### DISCUSSION TITLE
第5节课作业(17班)

### DISCUSSION THREAD

**Discussion by Jianfeng777** [InternLM/Tutorial#302] [作业]
请将第五节课作业上传到这里哦！

**Comment by StarDuster** [comment]
https://gist.github.com/StarDuster/a80cb2fa2c2b8285349822d7afee35d4

**Comment by fine-lab** [comment]
https://blog.csdn.net/SmallZCan_t/article/details/135608034?spm=1001.2014.3001.5501

**Comment by l241025097** [comment]
第五节课作业
[05LMDeploy大模型量化部署实践.pdf](https://github.com/InternLM/tutorial/files/13981533/05LMDeploy.pdf)

**Comment by AxrLH** [comment]
5 class:
https://blog.csdn.net/Axr_Samuel/article/details/135661888?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135661888%22%2C%22source%22%3A%22Axr_Samuel%22%7D

**Comment by futtttttttttttt** [comment]
https://zhuanlan.zhihu.com/p/678770743

**Comment by AxrLH** [comment]
作业：
https://blog.csdn.net/Axr_Samuel/article/details/135708523?csdn_share_tail=%7B%22type%22%3A%22blog%22%2C%22rType%22%3A%22article%22%2C%22rId%22%3A%22135708523%22%2C%22source%22%3A%22Axr_Samuel%22%7D

**Comment by zcg-joker** [comment]
[第五节课作业](https://blog.csdn.net/m0_55764641/article/details/135720626)
